docs/book/src/developer/providers/v1alpha4-to-v1beta1.md

```diff
@@ -50,3 +50,10 @@ v1alpha4, the same conversions have to be implemented.
 - **GetControlPlaneMachinesFromList** has been removed in favor of `FromMachineList(machines).Filter(collections.ControlPlaneMachines(cluster.Name))` in the util/collection package. 
 - **GetCRDMetadataFromGVK** has been removed in favor of `GetGVKMetadata`.
 
+## :warning: LeaderElectionResourceLock change :warning:
+
+The v1beta1 release defaults to use "leases" instead of "configmpasleases" as the LeaderElectionResourceLock for all managers leader election including the core controllers, bootstrap and control plane kubeadm and the Docker provider.
```

Comment:
```suggestion
The v1beta1 release uses "leases" instead of "configmapsleases" as the LeaderElectionResourceLock for all managers leader election including the core controllers, bootstrap and control plane kubeadm and the Docker provider.
```
nit, it's more like.. hard-coded :)